pkg/healthcheck/healthcheck_multicluster.go

```diff
@@ -152,6 +152,54 @@ func (hc *HealthChecker) multiClusterCategory() category {
 					return hc.checkDaisyChains()
 				},
 			},
+			{
+				description: "all mirrored services have endpoints",
+				hintAnchor:  "l5d-multicluster-services-endpoints",
+				warning:     true,
+				check: func(ctx context.Context) error {
+					return hc.checkIfMirroredServicesHaveEndpoints()
+				},
+			},
+			{
+				description: "all gateway mirrors have endpoints",
+				hintAnchor:  "l5d-multicluster-gateways-endpoints",
+				warning:     true,
+				check: func(ctx context.Context) error {
+					return hc.checkIfGatewayMirrorsHaveEndpoints()
+				},
+			},
+			{
+				description: "matching gateway mirrors of all mirrored services exist",
+				hintAnchor:  "l5d-multicluster-mirror-gateways",
+				warning:     true,
+				check: func(ctx context.Context) error {
+					return hc.checkIfAllMirrorServicesHaveGatewayMirrors()
+				},
+			},
+			{
+				description: "all gateways have external Ips",
```

Comment:
It might be nice to somehow make it clearer that the above 3 checks are for the cluster as a source and the next 3 checks are for the cluster as a target.  Not sure what the best way to convey that is, though.